### PR TITLE
Kanvi/implement any op

### PR DESCRIPTION
**Pull request by kanvi-nervana**
Implement op: Any

### CODE REVIEW

**Comment by kanvi-nervana** [comment]
TESTNOW

File: test/opexecuter.cpp [hunk 1]
```diff
@@ -160,6 +160,9 @@ void OpExecuter::CompareNGraphAndTF() {
       case DT_INT64:
         AssertTensorEquals<int64>(tf_outputs_[i], ngraph_outputs_[i]);
         break;
+      case DT_BOOL:
+        AssertTensorEquals<bool>(tf_outputs_[i], ngraph_outputs_[i]);
+        break;
       default:
```
Comment (sayantan-nervana) [inline test/opexecuter.cpp]: This would get added once 'All' goes in

Comment (kanvi-nervana) [reply test/opexecuter.cpp]: Yes, I know but I needed it there to work on the op :)

**Comment by kanvi-nervana** [comment]
TESTNOW

**Comment by kanvi-nervana** [comment]
TESTNOW

**Pull request merged**